src/main/java/frc/trigon/robot/subsystems/ledstrip/LEDStrip.java

```diff
@@ -0,0 +1,146 @@
+package frc.trigon.robot.subsystems.ledstrip;
+
+import com.ctre.phoenix.led.*;
+import edu.wpi.first.wpilibj.util.Color;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+public class LEDStrip extends SubsystemBase {
+    private static final CANdle CANDLE = LEDStripConstants.CANDLE;
+    private static int lastCreatedLEDStrip = 0;
```

Comment:
This is static, so it should be all caps
Also, this is the last created led stip _id_